global.json

```diff
@@ -1,8 +1,13 @@
 {
   "tools": {
-    "dotnet": "2.1.401"
+    "dotnet": "3.0.100",
+    "runtimes": {
+      "dotnet": [
+        "2.1.11"
```

Comment:
Is this for the xunit test runner?

Reply:
Actually good question. @jmarolf could we maybe just delete this and automatically use the runtime included with 3.0.100 SDK?

Reply:
yep

Reply:
Will probably do this in a future PR